docs/coding-agent/plans/active/issue-378-html5-owner-replace-position-plan.md

```
@@ -0,0 +1,132 @@
+# Plan: Issue 378 HTML5 Owner Replace Position
+
+- status: in_progress
+- generated: 2026-06-30
+- last_updated: 2026-06-30
+- work_type: code
+
+## Goal
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)